### ISSUE TITLE
Typo's in AbstractL4DStats

### ISSUE THREAD

**Issue by txdv** [koraktor/steam-condenser#68]
```
  this.favorites.put("campagin", favoritesNode.getElementsByTagName("campaign").item(0).getTextContent());
  this.favorites.put("campaginPercentage", Integer.parseInt(favoritesNode.getElementsByTagName("campaignpct").item(0).getTextContent()));
```

campagin != campaign

**Comment by koraktor** [comment]
Good to know. ;) Thanks.

**Comment by koraktor** [comment]
Java: Fixed several shortcomings in AbstractL4DStats

Closed by 2207abc44e19fcc78c8ec0cd4f2eb1afeec8b12f
